idseq_dag/steps/generate_alignment_viz.py

```diff
@@ -325,7 +327,12 @@ def get_sequences_by_accession_list_from_s3(accession_id_groups,
         for t in threads:
             t.join()
         if error_flags:
-            raise RuntimeError("Error in getting sequences by accession list.")
+            # Sequences might fail to be fetched if those accessions have been removed from nt_db.
+            # We can still proceed even if they failed to be fetched.
+            if error_flags["error"] == ERROR_MISSING_ACCESSION:
+                log.write("Some accessions could not be found in nt_db.")
+            else:
+                raise RuntimeError("Unknown error in getting sequences by accession list.")
```

Comment:
might be easier to add a more explicit message for the Datadog alert, like `SequenceByAccessionListError`

Reply:
I was actually planning on logging on `ntDbIndexError`, since that one doesn't fail the phylo tree job, whereas this one does. I can also give this one a more explicit name, but not sure I will set an alert on this one (it will just appear in the cloudwatch logs)